src/playbook.yaml

```diff
@@ -1,12 +1,49 @@
+# mostly following instructions from
+# https://docs.aws.amazon.com/AmazonECS/latest/developerguide/docker-basics.html
+
 - hosts: all
   become: yes
 
   tasks:
-    - name: install system packages
-      shell: "yes | aptdcon --hide-terminal --install 'build-essential'"
+    - name: Upgrade all packages
+      yum: name=* state=latest
+
+    - name: Install system packages
+      ansible.builtin.yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - "curl"
+        - "wget"
+        - "jq"
+        - "git"
+        - "amazon-linux-extras"
+        - "docker"
+
+    - name: Start docker service
+      ansible.builtin.service:
+        name: docker
+        state: started
```

Comment:
why must you start it during the build? what does it do/make?

Reply:
not sure but i think i was just following this yahoo's instructions.. https://github.com/Sage-Bionetworks-IT/packer-amazonlinux-docker/pull/1/files#diff-606e9b44b66f176373a6e9b820e9faf2a705817c3ac893deca167f7abee58b13R2